common/src/main/java/us/myles/ViaVersion/util/CollectionsUtil.java

```diff
@@ -0,0 +1,48 @@
+package us.myles.ViaVersion.util;
+
+import lombok.experimental.UtilityClass;
+import us.myles.ViaVersion.api.Via;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.util.HashMap;
+import java.util.Map;
+
+@UtilityClass
+public class CollectionsUtil {
+    private static Constructor<?> fastUtilLongObjectHashMap;
+    private static Constructor<?> fastUtilIntObjectHashMap;
+
+    static {
+        try {
+            fastUtilLongObjectHashMap = ReflectionUtil.getClass("it.unimi.fastutil.longs.Long2ObjectOpenHashMap").getConstructor();
```

Comment:
Should try it.unimi.*dsi*.fastutil
I suggest a code like ```try {... it.it.unimi.dsi.fastutil ...} catch ... {} try {... org.bukkit.craftbukkit ...} catch ... {}```

Reply:
It is better to throw an exception at the end, so that the rest of the classes in the static block CollectionsUtil.java did not try to load